Packs/Lumu/Integrations/Lumu/README.md

```diff
@@ -0,0 +1,7928 @@
+                        "C2C",
+                        "Malware"
+                    ],
+                    "contacts": 36,
+                    "description": "Malware family TaurusStealer",
+                    "hasPlaybackContacts": false,
+                    "id": "d39f4230-7a43-11ec-843d-dd7e2ea288b6",
+                    "labelDistribution": {
+                        "0": 2,
+                        "1851": 13,
+                        "1885": 20,
+                        "280": 1
+                    },
+                    "lastContact": "2022-01-27T16:10:09Z",
+                    "status": "muted",
+                    "statusTimestamp": "2022-02-08T20:40:31.481Z",
+                    "timestamp": "2022-01-20T22:53:47.347Z",
+                    "totalEndpoints": 3,
+                    "unread": false
+                },
+                {
+                    "adversaries": [
+                        "www.credi-familialtda.com"
+                    ],
+                    "adversaryId": "www.credi-familialtda.com",
+                    "adversaryTypes": [
+                        "Phishing"
+                    ],
+                    "contacts": 1,
+                    "description": "Phishing domain",
+                    "hasPlaybackContacts": false,
+                    "id": "64ea9a30-796f-11ec-843d-dd7e2ea288b6",
+                    "labelDistribution": {
+                        "864": 1
+                    },
+                    "lastContact": "2022-01-19T21:32:55.282Z",
+                    "status": "muted",
+                    "statusTimestamp": "2022-01-24T14:21:11.017Z",
+                    "timestamp": "2022-01-19T21:33:08.307Z",
+                    "totalEndpoints": 1,
+                    "unread": false
+                },
+                {
+                    "adversaries": [
+                        "saner.com.au"
+                    ],
+                    "adversaryId": "saner.com.au",
+                    "adversaryTypes": [
+                        "C2C",
+                        "Malware"
+                    ],
+                    "contacts": 7,
+                    "description": "Malware family P2PZeuS",
+                    "hasPlaybackContacts": false,
+                    "id": "e37738d0-7b0f-11ec-b95a-431da32564f1",
+                    "labelDistribution": {
+                        "1851": 1,
+                        "1885": 1,
+                        "1988": 1,
+                        "218": 4
+                    },
+                    "lastContact": "2022-03-07T21:21:01Z",
+                    "status": "muted",
+                    "statusTimestamp": "2022-01-21T23:18:45.525Z",
+                    "timestamp": "2022-01-21T23:14:31.261Z",
+                    "totalEndpoints": 2,
+                    "unread": false
+                },
+                {
+                    "adversaries": [
+                        "tempmail.co"
+                    ],
+                   
```

Comment:
Please update the example for this command

Reply:
README was re generated with all new parameters and arguments updated in YML file